### ISSUE TITLE
Bug: Twitter RTs don't work

### ISSUE THREAD

**Issue by pierrearma** [project-error/npwd#691]
### Development Issue

Unsure

### Phone Version

Phone version: 1.2.6

FXServer version: 5402

Database version: `10.6.5-MariaDB`

Database tables configuration: `import.sql` of version 1.2.6 with `utf8mb4_general_ci`

### Issue Description

An error occurs when trying to RT a Tweet.

### Issue Reproduction

1. Open Twitter.
2. RT a Tweet.

### Evidence

Server side error:

```
Uncaught TypeError: Bind parameters must not contain undefined. To pass SQL NULL specify JS null
@npwd/resources/dist/server/server.js(33033,21): 
@npwd/resources/dist/server/server.js(33028,26): execute
@npwd/resources/dist/server/server.js(33479,18): 
@npwd/resources/dist/server/server.js(33370,42): 
node:internal/process/task_queues(78,11): processTicksAndRejections

TypeError: Bind parameters must not contain undefined. To pass SQL NULL specify JS null
```

Client side error:

![image](https://user-images.githubusercontent.com/46777166/160279926-1fcd97d8-cad9-4167-acfe-2bf901336530.png)

**Comment by itschip** [comment]
Will look into it later today. Thanks for reporting.

**Comment by itschip** [comment]
Fixed in 7e57f5c8ea1f571ecd362afef5eaa5e2aa080f42
